substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/reports/AnalysisReportsOptions.java

```diff
@@ -52,6 +53,15 @@ protected void onValueUpdate(EconomicMap<OptionKey<?>, Object> values, Boolean o
         }
     };
 
+    @Option(help = "Change the output format of the analysis call tree. See: Reports.md.")//
+    public static final EnumOptionKey<CallTreeType> PrintAnalysisCallTreeType = new EnumOptionKey<CallTreeType>(CallTreeType.TXT) {
```

Comment:
You are the first user of `EnumOptionKey` in the substratevm codebase. Have you checked that this works as expected (tested with both option values `TXT` and `CSV`?

Reply:
Interesting :) 

Yes, I checked with both values explicitly set and without explicitly setting the option.